test/cases/api.sh

```diff
@@ -467,6 +510,8 @@ function createReqFileAzure() {
   ]
 }
 EOF
+### debug
+  cat "$REQUEST_FILE"
```

Comment:
Left over debug line.  If you want to keep it then use echo or green_print to add some delimiters in the output so this can be easily found for future reference. Alternaively you may want to upload this as an artifact in GitLab CI. 

Note: when printing on the terminal GitLab seems to do a good job of masking secret credentials, not sure what will happen if we upload the file as artifact.

Reply:
Thanks @atodorov ! I also hesitate if I should left this debug info. But for safe I'd like to remove it. 
Only changed this part in this push:
-### debug
-  cat "$REQUEST_FILE"